docs/oracle/tech-architecture.md

```diff
@@ -3,72 +3,44 @@ title: Technical Architecture of the DVM
-small, shell contracts whose job is only to deploy financial contracts. If a contract creator is approved, it means
-that it has been reviewed to ensure that all contracts that it can deploy will pay their fees honestly.
+The Store's funds are to be used to buy back and burn tokens. Right now, the funds can be withdrawn by the contract admin to do this off-chain. In the future, this buyback may be accomplished on-chain as a part of the contract logic.
 
-When a contract creator deploys a new financial contract, it registers it with the Registry contract to signal its
-approval.
+## Voting
 
-## Registry
+The DVM is powered by voters. When a dispute is made, UMA tokenholders vote on the value of an approved price identifier at a given time. The relative weighting of votes is determined by how many UMA tokens each participant holds.
 
-The Voting contract uses the Registry to validate whether a financial contract should be allowed to make a price
-request.
+Once a request is received, a voting period begins. Voters follow a typical commit-reveal cycle to provide their votes to the Voting contract.
 
-The Registry maintains two lists:
+After the vote period ends, `getPrice()` can be called to retrieve the price that was resolved. The contract rewards the voters who voted with the majority by minting them new tokens proportionally to their vote share. Voters who did not vote or voted incorrectly do not receive newly minted tokens.
 
-- Approved contract creators: these contracts have permission to register new financial contracts in the Registry.
+## VotingToken
 
-- Approved financial contracts: these contracts have been registered by an approved contract creator and are allowed to
-  make price requests to the Voting contract.
+The VotingToken is an ERC20 contract used to contribute to UMA protocol decisions, such as voting on UMA Improvement Proposals (UMIPs), price requests, and disputes made to UMA's Data Verification Mechanism (DVM).
 
-## Store
+## UMA Tokenholders
 
-The Store contract helps contracts compute their fees and collects those fees from contracts. The Store's funds are to
-be used to buy back and burn tokens. Right now, the funds can be withdrawn by the contract admin to do this offchain.
-In the future, this buyback may be accomplished on-chain as a part of the contract logic.
+UMA's DVM is designed with an economic guarantee around the cost it would take to corrupt the oracle and the profit someone would receive. The DVM ensures the price to obtain 51% of UMA tokens is greater than the profit from corrupting the DVM, as measured by the collateral stored in UMA's financial contracts. This is achieved through an inflationary reward (currently 0.05% of total network token supply), distributed pro-rata by stake to voters who participate and vote correctly. As long as there is an honest majority, voters will vote correctly.
```

Comment:
I think this page is greatly improved. Not now, but in future I think it would greatly benefit from a step by step explanation of the different parts of the diagram along with the explanations that are currently broken down by contract/participant.

Reply:
I agree. I will include this in the next PR.